genvm/vm_test.go

```diff
@@ -1459,10 +1461,12 @@ func runTestCases(t *testing.T, tcs []templateTestCase, genTester func(t *testin
 	}
 }
 
-func testWallet(t *testing.T, defaultGasPrice int, template core.Address, genTester func(t *testing.T) *tester) {
+const defaultGasPrice = 1
```

Comment:
The `defaultGasPrice` was 20 before, why the change to 1? Also should this maybe be just different values for different tests?